sale_order_partner_available_delivery/models/sale_order.py

```diff
@@ -0,0 +1,12 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    def _compute_partner_invoice_id(self):
+        for order in self:
+            order.partner_invoice_id = order.partner_id
```

Comment:
I know this seems to be stupid (because we're fully overwriting previous behavior), but we should preserve super call:
```suggestion
        super()._compute_partner_invoice_id()
        for order in self:
            order.partner_invoice_id = order.partner_id
```